File: python/udemy/5-estrutura-repetitiva/loops_definidos/tabuada.py
# Ler um número inteiro N, daí mostrar na tela a tabuada de N para 1 a 10, conforme exemplo

n = int(input('Número da tabuada: '))

for i in range(10):
    produto = n * (i + 1)
    print(n, " x ", (i + 1), " = ", produto)

File: c/udemy/5-estruturas-repetitivas/loops-definidos/tabuada.c
// Ler um número inteiro N, daí mostrar na tela a tabuada de N para 1 a 10

#include <stdio.h>

int main() {

    int n, i;

    printf("Valor inteiro: ");
    scanf("%d", &n);

    for (i = 1; i <= 10; i++) {
        printf("\n%d x %d = %d", n, i, i * n);
    }

    return 0;

}